src/hooks/useBalance.ts

```diff
@@ -29,15 +30,14 @@ function useCall(apiRef: any, addressRef: Ref<string>) {
   const unsub: Ref<VoidFn | undefined> = ref();
 
   const updateAccountH160 = async (address: string) => {
-    if (!address) return;
     try {
       const web3 = await createWeb3Instance(currentNetworkIdx.value as TNetworkId);
```

Comment:
according to this, do I have to create the Web3 instance every time the page updates the h160 account?

Reply:
good catch. I refactored as creating web3instance at boot time, it can be accessed by globally.